src/components/molecules/NavList.tsx

```diff
@@ -31,34 +26,59 @@ export function NavList() {
     items.map(item => ({
       label: enumMappings[item].label,
       filter: item,
-      path: enumMappings[item].url, 
+      path: enumMappings[item].url,
     }));
 
-  const subheaderLinkItems: SubheaderLinksProps[] = [
-    { items: roles, navigate },
-    { items: genres, navigate },
-  ]
+  const combinedLinks: ListButtonProps[] = [
+    ...navLinks.map(item => ({
+      label: item.label,
+      path: item.route,
+    })),
+    ...generateButtons(roles),
+    ...generateButtons(genres),
+  ];
+
+  const secondNavLinkOrder = [
+    'Animation',
+    'Documentary',
+    'Director of Photography',
+    'Director & Producer',
+    'Series',
+  ];
+
+  const secondNavLinks = combinedLinks
+    .filter(item => secondNavLinkOrder.includes(item.label))
+    .sort((a, b) => secondNavLinkOrder.indexOf(a.label) - secondNavLinkOrder.indexOf(b.label));
```

Comment:
🔧 This feels like a lot of work to get a list of ordered `NavLinks`

Something like:

```typescript
const orderedLinks: (Genre | Role)[] = ['animation', 'documentary', 'director_of_photography', 'director_producer', 'series']
//...
function NavList(){
  // ...
  const secondNavLinks = orderedLinks.map(linkEnum => (
    <NavLinks
      key={`burger-link-to-${linkEnum}`}
      route={enumMappings[linkEnum].url
      label={enumMappings[linkEnum].label}
      onClick={() => navigate(enumMappings[linkEnum].url)}
    />
  )
  ...
}
```

Reply:
Ok, it did seem like a lot when I wrote it, but was also the solution I could get to today. Couldn't figure out a way to make this more concise. 

Your rewrite is great!

I'll figure this out tomorrow.